### DISCUSSION TITLE
Whats wrong with my external library?

### DISCUSSION THREAD

**Discussion by isgm570** [immich-app/immich#14289] [Q&A]
Ive managed to mount C:\Users\forendr\Desktop\pictures\picturea⁠ to /media `C:\Users\forendr\Desktop\pictures\picturea:/media/` in docker-compose.yml. Entered docker-compose up -d or whatever the command is, added external libraries C:\Users\forendr\Desktop\pictures\picturea⁠, and tried putting  a photo in this folder but nothing works

**Comment by schuhbacca** [comment]
You add the right side of the docker mount, after the colon. Not the part on the left.

**Reply by isgm570** [reply]
but isnt /media/ after the colon? What am i missing?

**Reply by bo0tzz** [reply]
You're entering the wrong path in the UI

**Reply by isgm570** [reply]
/usr/src/app/external from the guide doesnt work, `Error
Invalid import path: Path does not exist (ENOENT)
(Immich Server Error)`, abd /media allows me to save it but nothing happens
